app/public/locales/en/home.json

```diff
@@ -0,0 +1,5 @@
+{
+  "orangePill": "Orange pill the world around you.",
```

Comment:
It looks like these translations aren't used yet in this PR, are you able to consume them in the index file?

Reply:
I'll remove this file in this PR, and add it in a future.